components/Blog/SideNav.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react'
 
-function SideNav ({ blogSchools, selectedSchool, onSchoolSelected }) {
+function SideNav ({ schools, selectedSchool, onSchoolSelected }) {
   return (
-    <div className='side-nav-blog pa1'>
+    <div className='side-nav-blog flex flex-column w-40 self-center pa2'>
       {
-        blogSchools.map((school) =>
-          <a className='pa2 tc ttc' key={school} data-school={school} onClick={(event) => onSchoolSelected(event.target.getAttribute('data-school'))}>
-            {school}
-          </a>
-        )
+        schools.map((school) => {
+          const isCurrentSchoolSelected = school === selectedSchool
+          const className = `${isCurrentSchoolSelected ? 'selected-school' : ''} pa2 tc ttc mb1 bg-card`
+          return (
+            <a className={className} key={school} data-school={school} onClick={(event) => onSchoolSelected(event.target.getAttribute('data-school'))}>
```

Comment:
`event.target.getAttribute('data-school')` is not needed. Just pass the `school` prop since it's the same value as the `data-school` attribute.